code/standards/pmdruleset.xml

```diff
@@ -15,23 +15,60 @@
   <description>
   PMD rules customised for PCGen's code. 
   </description>
-  
-    <rule ref="rulesets/java/logging-java.xml" />
-    <rule ref="rulesets/java/basic.xml" />
-    <rule ref="rulesets/java/braces.xml" />
-    <rule ref="rulesets/java/codesize.xml" >
-     <!-- 
-        <exclude name="TooManyFields" />
-        <exclude name="TooManyMethods" />
-       -->
-    </rule>
-    
-  <rule ref="rulesets/java/unnecessary.xml">
-    <exclude name="UselessParentheses"/>
+
+  <!--<rule ref="rulesets/java/android.xml">-->
+  <rule ref="rulesets/java/basic.xml">
+    <exclude name="AvoidUsingHardCodedIP" />
+  </rule>
+  <rule ref="rulesets/java/braces.xml" />
+  <rule ref="rulesets/java/clone.xml">
+    <exclude name="ProperCloneImplementation" />
+    <exclude name="CloneThrowsCloneNotSupportedException" />
+  </rule>
+  <rule ref="rulesets/java/codesize.xml" >
+    <exclude name="TooManyFields" />
```

Comment:
Does this provide value? I've found that the number of fields is rarely an issue, but the complexity and breakdown is. I'll remove the exclusions though (this PR is still strictly better than where we are today)